README.md

```diff
@@ -204,6 +204,16 @@ bin/slider destroy presto1
 bin/slider create presto1 --template appConfig.json --resources resources.json
 ```
 
+# 'Flex'ible app
+
+Flex the number of Presto workers to the new value. If greater than before, new copies of the  worker will be requested. If less, component instances will be destroyed.
+
+Changes are not immediate and depend on the availability of resources in the YARN cluster. Make sure while flex that there is an extra free node(if adding) available with no existing presto component. Otherwise Slider will add Presto workers to the existing node where Presto might already be running and deploying the newly 'flex'ed worker component will fail.
+
+```
+bin/slider flex presto1 --component WORKER 2 # eg: if there is already a WORKER component node, this will deploy one more worker node assuming you have atleast 1 free node and other resources on your cluster where you need to deploy Presto WORKER
```

Comment:
As I understand this command works differently to what you wrote here. 
It is going to set 2 instances of workers, so if you had 1 before then it will try to set up a new one, if you had 2 then nothing needs to be done and if you had 3 nodes then one node will be stopped.
Am I right, if so please fix command description.